packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -36,7 +36,10 @@ export async function activate(context: ExtensionContext): Promise<void> {
   const clientOptions: LanguageClientOptions = {
     documentSelector: [{ scheme: "file", pattern: "**/*.{view,fragment}.xml" }],
     synchronize: {
-      fileEvents: workspace.createFileSystemWatcher("**/*.{view,fragment}.xml"),
+      fileEvents: [
+        workspace.createFileSystemWatcher("**/*.{view,fragment}.xml"),
```

Comment:
do we still need this `.xml` file event? because we don't seem to handle `manfiest.json` in `onContentChange`?